### DISCUSSION TITLE
论坛寄了，问了几个朋友都上不去QAQ

### DISCUSSION THREAD

**Discussion by moushicheng** [mamoe/mirai#2817] [Q&A]
<img width="559" alt="image" src="https://github.com/mamoe/mirai/assets/54257412/75ccabc5-b6c1-4317-841c-ed603162e70e">

**Comment by MrXiaoM** [comment]
mamoe.net是17年注册的，没实名，现在被拒绝解析了，等实名完就能访问了，域名商效率极低。
临时解决方法是改 host，缺点是不能挂梯子
```
52.74.27.102 mirai.mamoe.net
185.199.109.153 docs.mirai.mamoe.net
185.199.109.153 kdoc.mirai.mamoe.net
2406:da18:976:8b00:c1b7:4799:ba69:db29 mirai.mamoe.net
2606:50c0:8001::153 docs.mirai.mamoe.net
2606:50c0:8001::153 kdoc.mirai.mamoe.net

```

**Reply by zhaodice** [reply]
把域名转到namesilo啊

**Comment by Takeoff0518** [comment]
> 把域名转到namesilo啊

>注册商设置禁止删除(clientDeleteProhibited)
>注册商设置暂停解析(clientHold)
>注册商设置禁止转移(clientTransferProhibited)
